tests/unit/parts/test_yaml_utils.py

```diff
@@ -128,6 +128,19 @@ def test_yaml_load_not_core22_base():
     assert str(raised.value) == "base is core20"
 
 
+def test_yaml_load_esm_base():
+    with pytest.raises(errors.MaintenanceBase):
+        yaml_utils.load(
+            io.StringIO(
+                dedent(
+                    """\
+            base: core
```

Comment:
can you fix this weird formatting here?

Reply:
can I do this in a separate PR, or try? As it is the same formatting as the rest of this module